src/pushprocessor.ts

```diff
@@ -90,6 +90,73 @@ const DEFAULT_OVERRIDE_RULES: IPushRule[] = [
         ],
         actions: [],
     },
+    {
+        // Matrix v1.7
+        rule_id: ".m.rule.is_user_mention",
+        default: true,
+        enabled: true,
+        conditions: [
+            {
+                kind: ConditionKind.EventPropertyContains,
+                key: "content.m\\.mentions.user_ids",
+                value: "[the user's Matrix ID]",
+            },
+        ],
+        actions: [
+            PushRuleActionName.Notify,
+            {
+                set_tweak: TweakName.Sound,
+                value: "default",
+            },
+            {
+                set_tweak: TweakName.Highlight,
+            },
+        ],
+    },
```

Comment:
This needs slotting in before, rather than after, a bunch of existing rules. I think we need more intelligence than just appending the new rules to the existing list.